src/node/desktop/src/core/file-path.ts

```diff
@@ -672,11 +672,12 @@ export class FilePath {
    * Checks whether this file path is a directory.
    */
   isDirectory(): boolean {
-    const stat = fs.lstatSync(this.path, {
-      throwIfNoEntry: false,
-    });
-
-    return stat != null && stat.isDirectory();
+    try {
+      const stats = fs.statSync(this.path);
```

Comment:
Note: I'm using `stat` instead of `lstat` here now, since I think we would want to resolve things like symbolic links here?